flatland_server/src/model.cpp

```diff
@@ -106,6 +106,18 @@ void Model::LoadBodies(YamlReader &bodies_reader) {
   } else {
     for (int i = 0; i < bodies_reader.NodeSize(); i++) {
       YamlReader body_reader = bodies_reader.Subnode(i, YamlReader::MAP);
+      if (!body_reader.Get<bool>("enabled", "true")) {
+        ROS_WARN_STREAM("Body "
+                        << Q(name_) << "."
+                        << body_reader.Get<std::string>("name", "unnamed")
+                        << " disabled");
+        continue;
+      } else {
```

Comment:
Just a style preference, we don't really need else here if we have continue above.